src/services/cache-config/CacheConfigService.ts

```diff
@@ -374,7 +386,6 @@ export class CacheConfigService {
     this.updateHelper(builder, address.concat("partition-handling", "PARTITION_HANDLING"), config["partition-handling"]);
     this.updateHelper(builder, address.concat("transaction", "TRANSACTION"), config.transaction);
     this.updateHelper(builder, address.concat("state-transfer", "STATE_TRANSFER"), config["state-transfer"]);
-    this.updateHelper(builder, address.concat("backup", "BACKUP"), config.backup);
```

Comment:
Right right, I got it now that I focused on the overall issues rather than just the code changes. All good @ryanemerson